PrizeDraw/MainWindow.xaml.cs

```diff
@@ -18,6 +19,8 @@ public partial class MainWindow : Window
         private const int WinnerTileTargetWidth = 800;
         private const int WinnerTileTargetHeight = 500;
         private readonly MainWindowViewModel _viewModel;
+        private static string AppFolder => Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData), "PrizeDraw");
+        private static string imageFolder => Path.Combine(AppFolder, "Images");
```

Comment:
I was just reusing the same code used to load the profile images and the 'default' image that we've been using (unless I've fundamentally misunderstood that code). I'll take a look at moving them back in with the application.

I can capitalise `ImageFolder`, I've always tended to use camel-case for private, but happy to change that.

Reply:
R# (and I'm assuming VS) suggest that static properties should be upper camel-case. For non-static, it would be camel-case with an underscore prefix. I really need to add an editorconfig file to the project.
However, _both_ of `AppFolder` and `imageFolder` are redundant, and don't do anything - so both can be removed.

Reply:
> I was just reusing the same code used to load the profile images and the 'default' image that we've been using (unless I've fundamentally misunderstood that code)

Ah, I was misreading it because both of your `AppFolder` and `imageFolder` are redundant here - and aren't used. Ie. when you had the unused `imageFolder` being passed into `LoadWinnerImage` and `LoadNoshowImage` - it looked liked your winner and noshow images were coming from the AppFolder (which wouldn't be right). But because you weren't using the `imageFolder` parameter, it's fine - your `@"Images/noshow.png"` will be looking in the exe folder, as it should do.

Reply:
So, TL;DR version is just to delete the unused `AppFolder` property, and all's good :)